docs/development/tests.rst

```diff
@@ -23,30 +23,17 @@ environments by running:
 
     tox
 
-By default, tox won't run tests from search,
-in order to run all test including the search tests,
-you need to override tox's posargs.
-If you don't have any additional arguments to pass,
-you can also set the ``TOX_POSARGS`` environment variable to an empty string:
-
-.. prompt:: bash
-
-    TOX_POSARGS='' tox
-
-.. note::
-
-   If you need to override tox's posargs, but you still don't want to run the search tests,
-   you need to include ``-m 'not search'`` to your command:
+To target a specific environment:
 
 .. prompt:: bash
 
-       tox -- -m 'not search' -x
+    tox -e py36
```

Comment:
Are these directions correct? It seems that we are telling people to run `tox` from _outside_ the container, but we don't really support that. I think we should update the whole guide.